api-spec/client/openapi/redirect/redirect-api.yaml

```diff
@@ -0,0 +1,218 @@
+          $ref: '#/components/schemas/PagopaIdTransaction'
+        idPsp:
+          description: PSP identifier
+          type: string
+        amount:
+          $ref: '#/components/schemas/AmountEuroCents'
+        urlBack:
+          description: pagoPA platform URL where redirect users after payment transaction have been completed
+          type: string
+          format: uri
+        description:
+          description: Payment description
+          type: string
+        paymentMethod:
+          description: Redirect payment method type
+          type: string
+          enum:
+            - BANK_ACCOUNT
+        paName:
+          description: Name of the payment notice issuer
+          type: string
+          minLength: 1
+          maxLength: 70
+        idPaymentMethod:
+          description: Unique id of the payment method chosen by the user
+          type: string
+          format: uuid
+      required:
+        - idTransaction
+        - idPsp
+        - amount
+        - urlBack
+        - description
+        - paymentMethod
+      example:
+        idTransaction: 3fa85f6457174562b3fc2c963f66afa6
+        idPsp: idPsp
+        amount: 99999999
+        urlBack: https://psp.site/payment
+        description: payment description
+        paymentMethod: BANK_ACCOUNT
+        paName: paName
+        idPaymentMethod: f40ccb17-3b0b-43d2-942a-2686eb121c6a
+    RedirectUrlResponse:
+      description: Redirect URL response
+      type: object
+      properties:
+        url:
+          description: URL where user has to be redirect to start payment
+          type: string
+          format: url
+        idTransaction:
+          $ref: '#/components/schemas/PagopaIdTransaction'
+        idPSPTransaction:
+          description: Unique identifier for the payment operation PSP side
+          type: string
+        amount:
+          $ref: '#/components/schemas/AmountEuroCents'
+        timeout:
+          description: Max time that pagoPA systems have to await for an outcome callback call. Any request received after this timeout will be discarded and transaction treated as for a missing callback call
+          type: integer
+          format: int32
+          minimum: 0
+          maximum: 600000
+      required:
+        - url
+        - idTransaction
+        - idPSPTransaction
+        - amount
+    AmountEuroCents:
+      description: Amount for payments, in euro cents
+      type: integer
+      format: int32
+      minimum: 0
+      maximum: 99999999
+    PagopaIdTransaction:
+      description: Uniquely identify a transaction
+      type: string
+      minLength: 32
+      maxLength: 32
+      example: 3fa85f6457174562b3fc2c963f66afa6
+  requestBodies:
+    RedirectUrlRequest:
+      required: true
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/RedirectUrlRequest'
+  securitySchemes:
+    PspApiKeyAuth:
+      type: apiKey
+      in: header
+      name: x-api-key
```

Comment:
TO DO: add refs to "nuova connettività" (and update api header `ocp-apim-subscription-key`)

Reply:
Ok, seems legit. Just one doubt, this way PSP implementation will expect api key to be present into ocp-apim-subscription-key header? Or this modification affers to api exposed by us to PSP (aka the outcome request)?

Reply:
Shouldn't this header be only on the API interface exposed on `pagopa-api`?

Reply:
@infantesimone merging this pr as is for the api key header name, will be updated in the future after integration clarification